content/issues/2022-06-13-repo-slow.md

```diff
@@ -0,0 +1,15 @@
+---
+title: JFrog's Artifactory slow (repo.jenkins-ci.org)
+date: 2022-06-13T08:30:00-00:00
```

Comment:
```suggestion
date: 2022-06-11T08:30:00-00:00
```

Reply:
? we are the 13 of june

Reply:
Slowness is from at least this weekend

Reply:
datadog disagree on that : https://jenkins.datadoghq.com/monitors/7375820?q=url%3Ahttps%3A%2F%2Frepo.jenkins-ci.org%2Fapi%2Fsystem%2Fping&live=1d